lib/grape/validations/validators/values_validator.rb

```diff
@@ -68,6 +68,17 @@ def check_excepts(param_array)
           param_array.none? { |param| excepts.include?(param) }
         end
 
+        def validate_proc(proc, param_array)
+          case proc.arity
+          when 0
+            param_array.all? { |_param| proc.call }
+          when 1
+            param_array.all? { |param| proc.call(param) }
+          else
+            raise ArgumentError, 'proc arity must be 0 or 1'
```

Comment:
Do we get some kind of useful call stack with a line number such as that the developer knows where the error comes from?

Reply:
Yeah, considering the ruby and rails versions that grape supports, the stack trace will show every sequence leading to this raise statement.